audio-worklet/src/bitcrusher-demo.js

```diff
@@ -124,13 +150,13 @@ class BitcrusherDemo {
    */
   switchBitcrusherNode() {
     // The function is called after the checkbox switches state.
-    const changeToScriptProcessor = !this.useAudioWorkletCheckBox_.checked;
-    if (changeToScriptProcessor) {
-      this.bitcrusherScriptProcessor_.output.connect(this.masterGain_);
-      this.bitcrusherAudioWorklet_.disconnect();
-    } else {
+    const changeToAudioWorklet = !this.useScriptProcessorCheckBox_.checked;
```

Comment:
Sorry. The review system is really convoluted because I didn't see the message in the review stream.

Still, you could have
```
const changeToScript = this.useScriptProcessorCheckBox_.checked;
if (change) {
  // script stuff
} else {
  // worklet stuff
}
```
Isn't this equivalent?